### PR TITLE
Display child info page

### LINKED ISSUE #13: Create Icons
### User Story
As a developer I need icons! I will build them in storybook. 

### Clarification
##To do
Create icons for different pages:
- [ ] Birthday
- [ ] cloth
- [ ] shoes/footprint
- [ ] plus
- [ ] pen
- [ ] go back
- [ ] X
- [ ] child
- [ ]  house

## Criteria
- [ ] Use design prototype
- [ ] Use storybook to build the components
- [ ] different colors

### Material | Images
Design:
https://xd.adobe.com/view/7f86dc69-703e-48db-70d4-8d3d07276926-09b7/

icons examples:
image

### PR DESCRIPTION
**Pull request by meloneminze**
In this pull request: 
- add routing for new pages
- split two components in two new one to use them at create and display pages
- add display pages and story
- modified components

close #13

### CODE REVIEW

File: client/src/components/CreateMemberInfoInput.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  border-radius: 20px;
+  border: none;
+  padding: 16px;
+  outline: none;
+  box-shadow: 0px 3px 6px;
+`;
+
+const CreateHeadline = styled.input`
+background-color: ${(props) => props.theme.colors.input};
+color: ${(props) => props.theme.colors.textSecondary}
```
Comment (lmachens) [inline client/src/components/CreateMemberInfoInput.js:17]: The formatting of this file is not working correctly. I guess it's related to a missing semicolon.

```suggestion
color: ${(props) => props.theme.colors.textSecondary};
```

File: client/src/components/CreateMemberInfoInput.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        placeholder="Adresse der Schule"
+        minLength="1"
+        maxLength="50"
+      ></CreateHeadline>
+      <CreateTextarea
+        placeholder="Aquariumstraße 42"
+        maxLength="200"
+        rows="3"
+        cols="45"
+      ></CreateTextarea>
```
Comment (lmachens) [inline client/src/components/CreateMemberInfoInput.js:53]:
```suggestion
      />
```

File: client/src/components/CreateMemberInfoInput.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+`;
+
+const CreateMemberInfoInput = () => {
+  return (
+    <CreateTextareaContainer>
+      <CreateHeadline
+        placeholder="Adresse der Schule"
+        minLength="1"
+        maxLength="50"
+      ></CreateHeadline>
```
Comment (lmachens) [inline client/src/components/CreateMemberInfoInput.js:47]:
```suggestion
      />
```

File: client/src/components/CreateMemberInfoInput.js (hunk 2 above)
Comment (meloneminze) [reply client/src/components/CreateMemberInfoInput.js:53]: 👍🏼

File: client/src/components/CreateMemberInfoInput.js (hunk 3 above)
Comment (meloneminze) [reply client/src/components/CreateMemberInfoInput.js:47]: 👍🏼

File: client/src/components/CreateMemberInfoInput.js (hunk 1 above)
Comment (meloneminze) [reply client/src/components/CreateMemberInfoInput.js:17]: Oh yes!

**Pull request merged**